src/importation/reader.js

```diff
@@ -0,0 +1,13 @@
+// On the browser side, code relies on the HTML5 File & FileReader API.
+// In tests, since run in a Node.js environment, `./__mocks__/reader.js` is used instead.
+
+export default {
+  read
+};
```

Comment:
Why is it necessary to export a nested object instead of the function itself?

Reply:
Having a single non-namespaced `read()` function in consumer code looked weird to me. But I can change it.